### PR TITLE
Added AFLOW-specific exercise 6

### PR DESCRIPTION
**Pull request by ctoher**
I added a 6th exercise focused on using OPTIMADE to search AFLOW using structural prototypes - I'll be going through these exercises during my presentation.

Best regards

Cormac.

### CODE REVIEW

**Comment by blokhin** [comment]
Isn't it a typo: `_aflow_aflow_prototype_label_relax`?

**Comment by ml-evs** [comment]
Awesome, thanks for this @ctoher. Hopefully other databases will add similar exercises. I will merge this and then restructure the exercise in the same format as the others (inside the Jupyter notebook).



> 
> 
> Isn't it a typo: `_aflow_aflow_prototype_label_relax`?

I think the field is specfically the AFLOW prototype (`aflow_prototype_label`) so with the `_aflow` prefix it looks weird but correct.

**Comment by ctoher** [comment]
Yes, the keyword in AFLOW is `aflow_prototype_label_relax`, so in OPTIMADE you add the `_aflow_` prefix and you end up with the double `aflow`. It looks weird, but when we came up with the new prototype labeling system, we already had a keyword called `prototype`, so the new one became `aflow_prototype`.

File: README.md [hunk 1]
```diff
@@ -359,6 +359,32 @@ Some potential prompts:
...
 
 <div class="cell markdown">
+    
+## Exercise 6
+
+</div>
+
+<div class="cell markdown">
+
+The AFLOW database is primarily built by decorating crystallographic prototypes,
```
Comment (JPBergsma) [inline README.md:369]: Perhaps you could still explain a bit what a crystallographic prototype is. 
People who have not followed the lecture may also do the tutorial.

**Pull request merged**